pagopa/io-app
# fix: [IABT-1348] Dismiss Zendesk views when app requires identification

## this PR depends on https://github.com/pagopa/io-react-native-zendesk/pull/20

## Short description
This PR introduces the ability (iOS only) to dismiss the Zendesk view when the app requires for user identification 
Without this the user, in a specific condition, could see the Zendesk view even when the app requires the user identification and when they come back inside the app, it appears frozen

More details in the ticket description 

https://user-images.githubusercontent.com/822471/156198940-ecbc2b61-9260-4d25-8a5a-59ace96ad0bf.mov


## How to reproduce the bug
- checkout `master` branch
- open a Zendesk View
- put the app in background
- wait for at least 30sec (to require identification)
- go back to the app
- you can see the Zendesk View
- when dismiss the Zendesk View, the app appears frozen

## How to test
- `yarn install; cd ios; pod install;`
- execute the same path of **How to reproduce the bug**
